.env.example

```
@@ -1,4 +1,5 @@
 POSTGRES_DB=p2p_shopping
 POSTGRES_USER=postgres
 POSTGRES_PASSWORD=postgres
-JWT_SECRET=your-secret-key-here-at-least-32-characters-long
\ No newline at end of file
+JWT_SECRET=your-secret-key-here-at-least-32-characters-long
+MONGO_URI=mongodb+srv://demouser:dreamteamE3@telemetrycluster.rd0vqir.mongodb.net/telemetry_db?retryWrites=true&w=majority&appName=TelemetryCluster
```

dotenv-linter (4.0.0)

[warning] [EndingBlankLine] No blank line at the end of the file

(EndingBlankLine)